android/src/main/java/com/amplitude/android/Configuration.kt

```diff
@@ -39,7 +39,9 @@ open class Configuration @JvmOverloads constructor(
     var locationListening: Boolean = true,
     var flushEventsOnClose: Boolean = true,
     var minTimeBetweenSessionsMillis: Long = MIN_TIME_BETWEEN_SESSIONS_MILLIS,
+    @Deprecated("Please use 'defaultTracking' instead.", ReplaceWith("defaultTracking.sessions"))
```

Comment:
nit: update comment to match
```suggestion
    @Deprecated("Please use 'defaultTracking.sessions' instead.", ReplaceWith("defaultTracking.sessions"))
```